erpnext_com/www/docs/user/manual/en/using-erpnext/articles/keyboard-shortcuts.md

```diff
@@ -0,0 +1,38 @@
+<!-- add-breadcrumbs -->
+#Keyboard Shortcuts
+
+ERPNext Version 12 is almost fully accessible using keyboard shortcuts.
+
+####Keyboard Shortcuts Dialog
+
+The keyboard shortcuts for any page can be viewed by opening the keyboard shortcuts dialog from the Help dropdown on the navbar.
+
+<img alt="Keyboard Shortcuts link" class="screenshot" src="{{docs_base_url}}/assets/img/articles/keyboard-shortcuts-link.png">
```

Comment:
add a red rectangle(default color) to highlight 'keyboard shortcuts'
crop this screenshot width wise, only a little padding on either side, view images in selling/buying pages for reference